src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,512 @@
+| PharmaceuticalDoseTime    | RECOMMENDED. Time of administration of pharmaceutical dose, relative to time zero (please see below). For an infusion, this should be a vector with two elements specifying the start and end of the infusion period. For more complex dose regimens, the regimen description should be complete enough to enable unambiguous interpretation of the DoseTime vector. Unit format of the specified pharmaceutical dose time should be seconds. |
+| Anaesthesia               | MAY. Details of anaesthesia used, if any.                                                                                                                                                                                                                                                                                                                                                                                                     |
+
+#### Radiochemistry
+
+This section is mandatory and contains information regarding the radioactive
+material used in the experiment.
+
+| Field name                    | Definition                                                                                                                                                                                                                                                                           |
+|-------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| InjectedRadioactivity         | REQUIRED. Total amount of radioactivity injected into the patient (for example, 400). Corresponds to DICOM Tag (0018,1074) Radionuclide Total Dose.                                                                                                                                  |
+| InjectedRadioactivityUnit     | REQUIRED. Unit format of the specified injected radioactivity (for example, "MBq").                                                                                                                                                                                                  |
+| InjectedMass                  | REQUIRED. Total mass of radiolabeled compound injected into subject (for example, 10). This can be derived as the ratio of the `InjectedRadioactivity` and `MolarRadioactivity`. *Note this is not required for an FDG acquisition, since it is not available, and can be set to -1. |
```

Comment:
I don't know how formal the BIDS specification is, but it seems to me that if you label something as `REQUIRED`, it *has* to be present. But if adding a note with some exclusions is fine. Otherwise, maybe this could be rephrased as
```suggestion
| InjectedMass                  | REQUIRED. Total mass of radiolabeled compound injected into subject (for example, 10). This can be derived as the ratio of the `InjectedRadioactivity` and `MolarRadioactivity`. *Note for an FDG acquisition this is not available and should be set to -1. |
```

Reply:
similar for entries below

Reply:
Dear @KrisThielemans, we completely agree and this is a great suggestion. We will accept this one and also change if for the other fields below. Kind regards, Mel